code/modules/atmospherics/machinery/components/trinary_devices/hypertorus.dm

```diff
@@ -408,7 +413,12 @@
 		return TRUE
 	return FALSE
 
-/obj/machinery/atmospherics/components/binary/hypertorus/core/process_atmos()
+/obj/machinery/atmospherics/components/binary/hypertorus/core/process()
+	if(next_slowprocess < world.time)
+		slowprocess()
+		next_slowprocess = world.time + 1 SECONDS //Set to wait for another second before processing again, we don't need to process more than once a second
```

Comment:
use the cooldown macros

Reply:
what are them? and i'm still unsure if this will be needed, i'm currently using it to slow down a bit the reaction to better control it

Reply:
Check `code/__DEFINES/cooldowns.dm`, and Ctrl+Shift+F for things like `COOLDOWN_DECLARE` in other files. They're just a standardized way of doing things like this.